lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.mjs

```diff
@@ -923,6 +923,77 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: [true],
+
+	accept: [
+		{
+			code: stripIndent`
+					a {
+						font-variant: foo foo foo foo foo foo foo;
+					}
+				`,
+		},
+	],
+
+	reject: [
+		{
+			code: stripIndent`
+					a {	
+						font-variant-ligatures: foo;
+						font-variant-position: foo;
+						font-variant-caps: foo;
+						font-variant-numeric: foo;
+						font-variant-alternates: foo;
+						font-variant-east-asian: foo;
+						font-variant-emoji: foo;
+						font-variant: foo;
```

Comment:
medium

Could you put real values?
When all values are `foo` the test cannot detect that the order is incorrect from the `fixed` string.
This is partly my fault.